blockchain/db/postgres/vr_transfers_db.py

```diff
@@ -0,0 +1,119 @@
+"""
+Copyright 2016 Disney Connected and Advanced Technologies
+
+Licensed under the Apache License, Version 2.0 (the "Apache License")
+with the following modification; you may not use this file except in
+compliance with the Apache License and the following modification to it:
+Section 6. Trademarks. is deleted and replaced with:
+
+     6. Trademarks. This License does not grant permission to use the trade
+        names, trademarks, service marks, or product names of the Licensor
+        and its affiliates, except as required to comply with Section 4(c) of
+        the License and to reproduce the content of the NOTICE file.
+
+You may obtain a copy of the Apache License at
+
+    http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the Apache License with the above modification is
+distributed on an "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY
+KIND, either express or implied. See the Apache License for the specific
+language governing permissions and limitations under the Apache License.
+"""
+
+__author__ = "Joe Roets, Brandon Kite, Dylan Yelton, Michael Bachtel, Alex Benedetto"
+__copyright__ = "Copyright 2016, Disney Connected and Advanced Technologies"
+__license__ = "Apache"
+__version__ = "2.0"
+__maintainer__ = "Joe Roets"
+__email__ = "joe@dragonchain.org"
+
+import psycopg2
+import psycopg2.extras
+import uuid
+
+from blockchain.qry import format_verification_record
+
+from postgres import get_connection_pool
+
+""" CONSTANTS """
+DEFAULT_PAGE_SIZE = 1000
+GET_VERIFIED_RECORDS = """SELECT * FROM vr_transfers"""
+SQL_MARK_RECORD = """UPDATE vr_transfers SET sent = B'1' WHERE verification_id = %s"""
+SQL_INSERT_QUERY = """INSERT INTO vr_transfers (
+                                  origin_id,
+                                  transfer_to,
+                                  verification_id
+                                ) VALUES (%s, %s, %s)"""
+
+
+def get_unsent_verification_records(node_transmit_id=None, verification_id=None):
```

Comment:
See comments below in the networking section. This method only needs the node_transmit_id param.